proto/auth.proto

```diff
@@ -49,3 +50,17 @@ message _LoginResponse {
     string text = 1;
   }
 }
+
+message _GenerateApiV1TokenRequest {
+  uint32 expiration_minutes = 1;
```

Comment:
I set this as `minutes` because I don't think we're going to need expiration at the second level granularity.